contents/BenefitAdvertisingCards.js

```diff
@@ -1,8 +1,21 @@
-import en from '../locales/en'
-const t = en
 const applyIcon = '/images/dashboard/apply-for-benefit-icon.svg'
 const estimateIcon = '/images/dashboard/estimate-retirement-income-icon.svg'
 
+const defaultDisplayFlags = {
```

Comment:
Have you considered creating property names that are representative of the program and type?
Something that works like TaskGroups https://github.com/DTS-STN/sc-digital-centre/blob/d44715a77e2713017d1b9ec108f1e98eb1777b10/contents/BenefitTasksGroups.js#L513

Reply:
Added a `typeCode` property to each adcard, and changed the defaultDisplayFlag object to use ProgramCodes and TypeCodes. Also added some extra TypeCodes for the various CPP subtypes that are advertised.